FieldSet: update demo description (#1852)
DevExpress/devextreme-demos: JSDemos/Demos/FieldSet/Overview/description.md

> DevExtreme comes with CSS classes that allow you to define the appearance of list-like structures that contain multiple field items - field sets.

Reviewer: ```suggestion
DevExtreme ships with CSS classes that allow you to define the appearance of list-like structures that contain multiple field items -- field sets.
```

## A Simple Field Set

> To create a basic field set, use the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) CSS class for the main container element. Next, use the [dx-fieldset-header](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset-header) class to specify the field set header. The header element can hold plain text, UI components, or custom markup. 

Reviewer: ```suggestion
To create a basic field set, use the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) CSS class for the main container element. Then, use the [dx-fieldset-header](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset-header) class to specify the field set header. The header element can contain plain text, UI components, or custom markup. 
```

> Create field set items as separate elements with the [dx-field](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field) class. The field element may include label and value elements. Define the label element by the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class. The value element uses the [dx-field-value-static](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-value-static) class and can display plain text or custom markup. 

Reviewer: > The field element **may** include label and value elements.

The word "may" holds a great degree of uncertainty. Use "can" instead.

I cannot say I understand this sentence:

> Define the label element by the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class.

Author: I tried to rewrite the last sentence:

To define the label element, use the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class.

or

Apply the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class to the label element.

Reviewer: I would have preferred the first variant. Thank you.

## A Field Set with DevExtreme Widgets

> You can also use DevExtreme UI components as the value elements. Specify the [dx-field-value](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-value) class in the component's HTML markup.

Reviewer: We haven't referred to "value elements" before.

```suggestion
You can also use DevExtreme UI components as value elements. Specify the [dx-field-value](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-value) class in the component's HTML markup.
```

## A Field Set with Custom Value Width

> To align all the values in the field set by width, attach a custom id to the element with the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) class and apply CSS rules according to the rules in the demo.

Reviewer: > all the values

This is a way too strong emphasis. Just "all values" is enough.

```suggestion
To align all values in the field set by width, attach a custom id to the element with the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) class and apply CSS rules according to the rules in the demo.
```

> To align all values in the field set by width, attach a custom id to the element with the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) class and apply CSS rules according to the rules in the demo.

Reviewer 2: `in the field set by width` Please explain

Author: I just realised there may be a confusion when I reread the sentence...
It should be read like this:

> To align all values in the **field set** by width, attach a custom id to the element with the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) class and apply CSS rules according to the rules in the demo.

So the "field set" is the name of the list structure, the values in this structure are "aligned by width". "Set" is not a verb here.
Maybe this variant will be more comprehensible:

> To align all field set elements by width, attach a custom id to the element with the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) class and apply CSS rules according to the rules in the demo.

Reviewer 2: Ok
